### PR TITLE
zcash_client_sqlite: Add `FsBlockDb::find_block`

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Review by daira — APPROVED** [review]
utACK (but I'm not familiar with the use case for this, so needs another review from someone who is).

File: zcash_client_sqlite/src/lib.rs [hunk 1]
```diff
@@ -1285,6 +1301,32 @@ mod tests {
...
+        let block_path = meta.block_file_path(&blocks_dir);
+
+        File::create(block_path)
+            .unwrap()
+            .write_all(&cb.encode_to_vec())
+            .unwrap();
+
+        meta
+    }
+
```
Comment (nuttycom) [inline zcash_client_sqlite/src/lib.rs:1329]: Should we actually make a method like this part of the `BlockStore` interface?

**Review by nuttycom — APPROVED** [review]
utACK

File: zcash_client_sqlite/src/lib.rs (hunk 1 above)
Comment (str4d) [reply zcash_client_sqlite/src/lib.rs:1329]: Maybe in future when we need it from Rust; for now this is logic that the mobile SDKs will be implementing themselves.

**Pull request merged**